exporters/otlp/all/src/test/java/io/opentelemetry/exporter/otlp/http/metrics/OtlpHttpMetricExporterBuilderTest.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.exporter.otlp.http.metrics;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.opentelemetry.sdk.common.export.MemoryMode;
+import io.opentelemetry.sdk.metrics.export.AggregationTemporalitySelector;
+import io.opentelemetry.sdk.metrics.export.DefaultAggregationSelector;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+public class OtlpHttpMetricExporterBuilderTest {
+
+  @Test
+  public void verifyToBuilderPreservesSettings() {
+    AggregationTemporalitySelector temporalitySelector =
+        Mockito.mock(AggregationTemporalitySelector.class);
+    DefaultAggregationSelector defaultAggregationSelector =
+        Mockito.mock(DefaultAggregationSelector.class);
+
+    OtlpHttpMetricExporter original =
+        OtlpHttpMetricExporter.builder()
+            .setMemoryMode(MemoryMode.IMMUTABLE_DATA)
+            .setAggregationTemporalitySelector(temporalitySelector)
+            .setDefaultAggregationSelector(defaultAggregationSelector)
+            .build();
+
+    OtlpHttpMetricExporter copy = original.toBuilder().build();
```

Comment:
Should clean exporters afterwards to free the thread pools initialized by the underlying http client libraries